### PR TITLE
feat(quest-details): replace play-streak bolts with sync progress

### PR DESCRIPTION
**Pull request by eliobricenov**

### CODE REVIEW

File: src/components/QuestDetails/components/StreakProgress/index.tsx [hunk 1]
```diff
@@ -152,14 +144,10 @@ export default function StreakProgress({
...
-        <div className={styles.progressIconsContainer}>
-          {lightningBoltIcons.map((filled, index) => (
-            <LightningBolt
-              className={classNames(styles.streakIcon, getFilledClass(filled))}
-              key={`lightningBolt:${index}`}
-            />
-          ))}
-          <LightningBolt />
+        <div className={styles.syncContainer}>
+          <Button type="secondaryGradient" onClick={onSync}>
```
Comment (BrettCleary) [inline src/components/QuestDetails/components/StreakProgress/index.tsx:148]: the size here is small in the design. 
image

Comment (BrettCleary) [inline src/components/QuestDetails/components/StreakProgress/index.tsx:148]:
```suggestion
          <Button type="secondaryGradient" onClick={onSync} size="small">
```

File: src/components/QuestDetails/components/StreakProgress/index.tsx [hunk 2]
```diff
@@ -152,14 +144,10 @@ export default function StreakProgress({
...
+            {i18n.sync}
```
Comment (BrettCleary) [inline src/components/QuestDetails/components/StreakProgress/index.tsx:149]: typography should be "button-sm"

File: src/components/QuestDetails/components/StreakProgress/index.tsx (hunk 1 above)
Comment (BrettCleary) [inline src/components/QuestDetails/components/StreakProgress/index.tsx:148]: the secondary gradient doesn't seem to match the designs. it's kind of small ui issue but I am wondering if we should update secondaryGradient to match the design or introduce a new gradient button type

**Review by BrettCleary — COMMENTED** [review]
my poor lightning bolts. i cri 😢 

Thanks for tackling this one! left some styling comments

File: src/components/QuestDetails/components/StreakProgress/index.tsx (hunk 1 above)
Comment (eliobricenov) [reply src/components/QuestDetails/components/StreakProgress/index.tsx:148]: we can update

**Comment by eliobricenov** [comment]
RIP lighting bolts

**Pull request merged**